draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -91,26 +103,30 @@ class GuidanceBlock extends React.Component<Props, State> {
         </div>
         <div
           className={classnames(styles.buttonContainer, {
-            [styles.secondaryAction]: secondaryAction,
+            [styles.secondaryAction]: secondary,
           })}
         >
-          <Button
-            label={button.label}
-            onClick={button.onClick}
-            icon={configureIcon}
-            iconPosition="end"
-          />
-          {secondaryAction && (
+          {primary && (
```

Comment:
I think the primary button always show, otherwise is a banner, not a guidance block